packages/connect-iframe/webpack/base.webpack.config.ts

```diff
@@ -167,7 +167,7 @@ export const config: webpack.Configuration = {
                 },
             }),
         ],
-        usedExports: false,
+        usedExports: true,
```

Comment:
I added `usedExports: false` in https://github.com/trezor/trezor-suite/pull/9896/files 
unfortunately, I don't remember why, maybe it is a leftover from some experiments. on the other hand it might have been something needed to make dynamic import of core work or maybe something with content script in webextension package. @martykan could you please check on this?

Reply:
It seems to work and it looks like it doesn't affect the output that much. 
But `core.js` is dynamically imported so maybe it makes sense to keep it.